ansible/README.md

```diff
@@ -0,0 +1,100 @@
+# Per team virtual workstation configuration
+## Why?
+Teams sometimes (or usually) need to follow instructions to set up their virtual workstations. 
+And some teams (maybe all of them) have manuals to follow. 
+This tiny project tries to automate that process and take away all setup hurdle from users.
+
+## When you add your team to `roles/teams` or edit existing team role
+Please, update `roles/common` folder: add any new common task that can be shared among other teams or can be used 
+by other users (as a role). This folder keeps all roles of all teams. When users can't find their team,
```

Comment:
Rephrase: When users.... -> If a user can't find their team, please create a new one on top of the common roles shared by all teams.